docs/en/setup/backend/backend-storage.md

```diff
@@ -65,9 +67,11 @@ SkyWalking rebuilds the ElasticSearch client on top of ElasticSearch REST API an
 correct request formats according to the server-side version, hence you don't need to download different binaries
 and don't need to configure different storage selectors for different ElasticSearch server-side versions anymore.
 
-For now, SkyWalking supports ElasticSearch 6.x, ElasticSearch 7.x, ElasticSearch 8.x, and OpenSearch 1.x, their
+For now, SkyWalking supports ElasticSearch 7.x, ElasticSearch 8.x, and OpenSearch 1.x, their
 configurations are as follows:
 
+_Notice, ElasticSearch 6 worked and is not promised due to end of life officially._
```

Comment:
"worked" sounds like it is not working NOW

Reply:
From my understanding of English(I could be wrong as I am not a native speaker)
- `may work` means there are possibilities of working, but not sure
- `might work` means less possible compared with `may work`
- `worked` means we tried, and it worked before. 

This is the reason I choose `worked` as the new statement.